src/xkbcomp/rules.c

```diff
@@ -694,20 +694,22 @@ match_group(struct matcher *m, struct sval group_name, struct sval to)
 
 static bool
 match_value(struct matcher *m, struct sval val, struct sval to,
-            enum mlvo_match_type match_type)
+            enum mlvo_match_type match_type, bool wildcard_matches_empty)
```

Comment:
how about something like this:
```c
enum wildcard_match_type {
   WILDCARD_MATCH_EMPTY,
   WILDCARD_MATCH_NONEMPTY,
};
```
and then the invocations would be more readable than a plain `true`/`false`.

Reply:
No boolean blindness? Good idea. Although it shines much more in public API.